betaflight/betaflight
# GPS rescue + New AltHold controller (not for merge)

Ready for testing. Test instructions:
1) Disable BARO (only for testing):
`set baro_hardware = NONE`
2) Enable new alt-hold mode:
`set gps_rescue_althold_mode = COOL` (for new one or DEFAULT for old one)
3) Go and test.

Expected behavior - slower reaction speed. Controller was optimized for ALT-hold mode and not for GPS rescue.
